tests/integration_tests/test_experiment.py

```diff
@@ -77,6 +80,34 @@ def test_experiment_seq_seq1(csv_filename):
         run_experiment(input_features, output_features, data_csv=rel_path)
 
 
+def test_experiment_seq_seq1_model_def_file(csv_filename, yaml_filename):
+    # seq-to-seq test to use model definition file instead of dictionary
+    input_features = ('[{name: utt, type: text, reduce_output: null, '
+                      'vocab_size: 10, min_len: 10, max_len: 10,'
+                      ' encoder: embed}]')
+
+    output_features = ('[{name: iob, type: text, reduce_input: null, '
+                       'vocab_size: 3, min_len: 10, max_len: 10,'
+                       ' decoder: tagger}]')
+
+    # Save the model definition to a yaml file
+    model_definition = yaml.load(model_definition_template.substitute(
+        input_name=input_features,
+        output_name=output_features
+    ))
+    yaml.dump(model_definition, open(yaml_filename, 'w'))
```

Comment:
Yep, I think it'd be safer to use `with`

Reply:
will fix this